www/js/main.js

```diff
@@ -53,10 +53,11 @@ var BB = {
             [null,      null,       null,       null,       null,       'silver',   null,       null,       null,       null]
         ];
         
+        var position = parseInt(BB.renderer.width * 0.5) - (5 * 30)
```

Comment:
@daydreamheart Thank you for fixing ... anyway, could you explain here a little bit what is the formula is from? like, what is `0.5`, and `5 * 30`?
Thank you

Reply:
It determines the position of the first block from the left size.

**parseInt(BB.renderer.width * 0.5)** - With this, we get the 1/2 size of the screen width

**(5 * 30)** 
 5 is the 1/2 of the total number of blocks in one line (blockMap). 
30 referred to the width of 1 individual block.

parseInt(BB.renderer.width * 0.5) - (5 * 30)
We get the starting position by subtracting and it will always center the element according to the screen width.